src/main/java/com/nekomaster1000/infernalexp/client/entity/render/ShroomloinRenderer.java

```diff
@@ -5,16 +5,21 @@
 import com.nekomaster1000.infernalexp.client.entity.model.ShroomloinModel;
 import com.nekomaster1000.infernalexp.entities.ShroomloinEntity;
 
+import net.minecraft.client.renderer.RenderType;
```

Comment:
Is there a reason for this new import? I can't see instantly a new thing added that needs this import.

Reply:
Huh that is kind of weird, but I'm pretty sure there are changes if you press into the commits

Reply:
It's possible while you were working in passing it got imported and then you removed what imported it. I would make sure that your IDE has turned on to "Remove Unused Imports" if it has that option.